powerlevel9k.zsh-theme

```diff
@@ -337,6 +337,47 @@ prompt_background_jobs() {
   fi
 }
 
+# Segment that indicates usage level of current partition.
+set_default POWERLEVEL9K_HDD_USAGE_ONLY_WARNING false
+set_default POWERLEVEL9K_HDD_USAGE_WARNING_LEVEL 90
+set_default POWERLEVEL9K_HDD_USAGE_CRITICAL_LEVEL 95
+prompt_hdd_usage() {
+  local current_state="unknown"
+  typeset -AH hdd_usage_forecolors
+  hdd_usage_forecolors=(
+    'normal'        'yellow'
+    'warning'       "$DEFAULT_COLOR"
+    'critical'      'white'
+  )
+  typeset -AH hdd_usage_backcolors
+  hdd_usage_backcolors=(
+    'normal'        $DEFAULT_COLOR
+    'warning'       'yellow'
+    'critical'      'red'
+  )
+
+  local level="${$(df -k . | sed -n '2p' | awk '{ print $5 }')%%\%}"
```

Comment:
Hmm. I was too tired yesterday as well.. Now I am too late, but this control structure should work:
local level="${$(df -k . | sed -n '2p' | awk '{ print $5 }')%%\%}"
```zsh
# Default state is "normal" unless POWERLEVEL9K_HDD_USAGE_ONLY_WARNING
# is set to true, then it is "unknown"
if [[ "$POWERLEVEL9K_HDD_USAGE_ONLY_WARNING" != true ]]; then
  current_state='normal'
fi

if [ "$level" -ge "$POWERLEVEL9K_HDD_USAGE_CRITICAL_LEVEL" ]; then
  current_state='critical'
elif [ "$level" -ge "$POWERLEVEL9K_HDD_USAGE_WARNING_LEVEL" ]; then
  current_state='warning'
fi
```